Frameworks/CoreFoundation/Base.subproj/CFBase.h

```diff
@@ -498,6 +498,7 @@ CF_INLINE CFRange CFRangeMake(CFIndex loc, CFIndex len) {
 CF_EXPORT
 CFRange __CFRangeMake(CFIndex loc, CFIndex len);
 
+#define NSMakeRangeFromCF(cfr) NSMakeRange(cfr.location == kCFNotFound ? NSNotFound : cfr.location, cfr.length)
```

Comment:
>NSMakeRange [](start = 31, length = 11)

I don't think this should be here; it introduces a dependency from CF headers to Foundation headers.